shuup/front/templates/shuup/front/maintenance.jinja

```diff
@@ -5,12 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" type="image/x-icon" href="{{ static("img/shuup.ico") }}">
-    <link href='//code.cdn.mozilla.net/fonts/fira.css' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="{{ static("shuup/front/css/style.css") }}">
+
     <title>{% trans %}Maintenance mode{% endtrans %}</title>
     <style>
         body {
-            font-family: "Fira Sans", sans-sefif;
+            font-family: "Open Sans", sans-sefif;
```

Comment:
do you still need this line? shoundl't this font-family com from `static("shuup/front/css/style.css")` ?

Reply:
As I understood it, this is how it's using the imported font from style.css